feathr_project/feathr/spark_provider/_localspark_submission.py

```diff
@@ -106,7 +108,23 @@ def submit_feathr_job(
                 print(python_files)
                 spark_args.append(python_files[0])
         else:
-            spark_args.extend(["--class", main_class_name, main_jar_path])
+            if not python_files:
```

Comment:
If a main jar is provided, we can use this main jar directly rather than add any dependencies.

Reply:
If a main jar is provided, we also need to consider if there's a python file or not. If there's a python file provided, we should still go thru the python path.

Reply:
But I guess maybe we don't need to add those dependencies?